articles/sentinel/sap/deploy-sap-btp-solution.md

```diff
@@ -69,14 +65,14 @@ To set up the BTP account and the solution:
 1. Select the resource group and the Microsoft Sentinel workspace in which to deploy the solution.
 1. Select **Next** until you pass validation, and then select **Create**.
 1. When the solution deployment is finished, return to your Microsoft Sentinel workspace and select **Data connectors**.
-1. In the search bar, enter **BTP**, and then select **SAP BTP (using Azure Function)**.
+1. In the search bar, enter **BTP**, and then select **SAP BTP**.
 1. Select **Open connector page**.
-1. On the connector page, make sure that you meet the required prerequisites and complete the configuration steps. In step 2 of the data connector configuration, specify the parameters that you defined in step 4 in this section.
+1. On the connector page, make sure that you meet the required prerequisites and complete the configuration steps. Click **Add account**.
+1. Specify the parameters that you defined in step 4 in this section. The subaccount name specified will be projected as a column in the SAPBTPAuditLog_CL table. This can be used to filter the logs when adding multiple subaccounts.
```

Comment:
```suggestion
1. Specify the parameters that you defined earlier during the configuration. The subaccount name specified is projected as a column in the `SAPBTPAuditLog_CL` table, and can be used to filter the logs when you have multiple subaccounts.
```

this isn't really clear to me - which is step 4? please check to see that this is correct.